ansible/playbooks/rhel.yml

```diff
@@ -35,6 +35,7 @@
         - bind-utils
         - cpio
         - cups-devel
+        - expat-devel
```

Comment:
Is this meant to have been moved?

Reply:
Yes, it was meant to be moved.  Previously it was only installed if rhel7 and ppcle were set and required as a git src compile.  Its a core dependency for XML::Parser so I moved expat to the generic install section.

Reply:
Tested, package is available on all archs and RHEL 6/7